metricbeat/docs/modules/docker.asciidoc

```diff
@@ -62,6 +62,9 @@ metricbeat.modules:
   # If set to true, replace dots in labels with `_`.
   #labels.dedot: false
 
+  # Docker module supports metrics collection from podman's docker compatible API. In case of podman set to true.
```

Comment:
Wondering if we should protect ourselves and verify somewhere the minimum version of podman we support or at least we tested with?

I see https://podman.io/blogs/2020/07/01/rest-versioning.html that api introduced in versions > v2.0
and probably in our tests we used the the [v2 API ](https://docs.podman.io/en/stable/_static/api.html?version=v4.0)version(inn versions >4.0)

Reply:
I guess this compatibility section needs to be updated: https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-docker.html#_compatibility_16